fuzz_lightyear/supplements/factory.py

```diff
@@ -19,9 +20,17 @@
 from fuzz_lightyear.util import listify_decorator_args
 
 
-def register_factory(keys: Union[str, Iterable[str]]) -> Callable:
+def _none_func() -> None:
+    return None
+
+
+def register_factory(
+    keys: Union[str, Iterable[str]],
+    endpoints: Union[str, Iterable[str]] = None,
```

Comment:
```suggestion
    *args,
    operation_ids: Optional[Union[str, Iterable[str]]] = None,
```

Reply:
Let's require `operation_ids` to be a keyword-only argument, for enforced readability. After all, it's not entire clear what this does:

```
@fuzz_lightyear.register_factory('biz_id', 'test_operation')
```